### PR TITLE
New version: yt-dlp.yt-dlp.nightly version 2024.01.29.232706

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/137205)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @Exorcism0666,

The package didn't pass a Defender or similar security scan. This might be a false positive and we can rescan tomorrow.

(Automated response - build 684.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978335187 2024-01-30 13:10:03.921 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerSharedLib\SHA256.cpp(144)\WindowsPackageManager.dll!00007FFFAEA9273F: (caller: 00007FFFAEADAEC4) Exception(1) tid(1a44) 8A150040  2024-01-30 13:10:09.205 [FAIL] Installer failed security check. Url: https://github.com/yt-dlp/yt-dlp-nightly-builds/releases/download/2024.01.29.232706/yt-dlp.exe Result: 0x80004005 Installation failed with exit code -1978334972

(Automated response - build 684.)

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

File: manifests/y/yt-dlp/yt-dlp/nightly/2024.01.29.232706/yt-dlp.yt-dlp.nightly.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Created with komac v2.0.2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: yt-dlp.yt-dlp.nightly
+PackageVersion: 2024.01.29.232706
+InstallerLocale: en-US
+InstallerType: exe
```
Comment (stephengillie) [inline manifests/y/yt-dlp/yt-dlp/nightly/2024.01.29.232706/yt-dlp.yt-dlp.nightly.installer.yaml:7]:
```suggestion
InstallerType: portable
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978334972

(Automated response - build 726.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978334972

(Automated response - build 726.)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**